src/vsr/free_set.zig

```diff
@@ -608,7 +623,7 @@ pub const FreeSet = struct {
     }
 
     /// Decodes the compressed bitset chunks in `source_chunks` into `target_bitset`.
-    /// Panics if the `source_chunks` encoding is invalid.
+    /// Panic if the `source_chunks` encoding is invalid.
```

Comment:
```suggestion
    /// Panics if the `source_chunks` encoding is invalid.
```

It's indicative, not imperative.